src/openloco/graphics/gfx.cpp

```diff
@@ -747,6 +772,19 @@ namespace openloco::gfx
         fill_rect_inset(dpi, x, y, x + dx - 1, y + dy - 1, colour, flags);
     }
 
+    // 0x00452DA4
+    void draw_line(gfx::drawpixelinfo_t* dpi, int16_t left, int16_t top, int16_t right, int16_t bottom, uint32_t color)
```

Comment:
```suggestion
    void draw_line(gfx::drawpixelinfo_t* dpi, point_t start, point_t stop, uint8_t color)
```

Reply:
Also: colour.

Reply:
`fill_rect`, `fill_rect_inset` and `draw_rect` have committed similar transgressions, heh. Will correct.